### DISCUSSION TITLE
REALL??!!

### DISCUSSION THREAD

**Discussion by Theoyeah** [DEMOLITIONDON96/Demolition-Engine#82] [Source Show & Tell]
https://user-images.githubusercontent.com/97792861/165960605-02e1aca4-111f-4355-85dd-feb2b5d53582.mp4

LEAKK ????!!!!

**Comment by ThatOneSillyGuyDon** [comment]
Hell is already a thing on my engine lol

**Reply by Theoyeah** [reply]
i know but... Imagine 5 FUCKIN DIFFICULTYS ???!!

**Reply by ThatOneSillyGuyDon** [reply]
> i know but... Imagine 5 FUCKIN DIFFICULTYS ???!!

Me, who has Crazy Difficulty (renamed to Erect soon) as an option for the remixes, and Mania for multikeys when we get them added in

**Reply by Theoyeah** [reply]
When Week 8 soure will be out, ill replace Hell difficulty to erect + fucked will be imported to remixes

**Reply by Theoyeah** [reply]
Even that the update would be just tomorrow, it is unknown when it will be out..

**Reply by ThatOneSillyGuyDon** [reply]
> Even that the update would be just tomorrow, it is unknown when it will be out..

It ain’t coming tomorrow, it got delayed, PhantomArcade announced that on Twitter last week
